Windows/PerfOptions.py

```diff
@@ -160,8 +161,8 @@ def accept(self):
             self.close()
         except ValueError as ve:
             PopUpWrapper("Invalid vector format", str(ve), None,
-                            QtWidgets.QMessageBox.Yes,
-                            QtWidgets.QMessageBox.No, None, quit)
+                            QMessageBox.Yes,
```

Comment:
Nothing wrong, I want to mention that we are not keeping any convention about that ;/